app-modules/engagement/src/Filament/Actions/RelationManagerSendEngagementAction.php

```diff
@@ -231,13 +244,13 @@ protected function setUp(): void
                             ->hidden(fn (Get $get): bool => $get('channel') === NotificationChannel::Sms->value)
                             ->helperText('You can insert recipient or your information by typing {{ and choosing a merge value to insert.')
                             ->columnSpanFull(),
-                        EngagementSmsBodyInput::make(context: 'create', form: $form),
```

Comment:
Have a look inside the `EngagementSmsBodyInput` and see how `$form` is just used to determine the value of `showMergeTagsInBlocksPanel`? I think we should still pass it in if we can